charts/datasaur/templates/main/ingress.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.ingress.host }}
```

Comment:
by adding if here, it means that we want host variable to be optional, is it expected?

Reply:
yes, if they only want to install llm labs, then ingress.host is not required